sites/shopify.js

```diff
@@ -114,7 +114,7 @@ async function checkout({
 
     const captchaSelector = 'div#g-recaptcha';
     try {
-      hasCaptcha = await page.waitForSelector(captchaSelector);
+      hasCaptcha = await page.waitForSelector(captchaSelector, {timeout: 2000});
```

Comment:
I understand reducing this from the default 30s timeout but I think we should do at least 5 seconds. Anecdotally, I've seen the captcha take a few seconds to load.

Reply:
cool thanks. i will amend this to 5 seconds

Reply:
In regards to your comments, it was just not typing the email fast enough.
It was waiting for the captcha but it never did appear. So it will only type the email after the async fails. 
I am using my residential IP address so that might be the reason why the captcha doesn't appear.

This might be more specific to this website that I might be targeting.

Reply:
@samc621 can you review this.
Sorry I misunderstood your question at first. Yes I've removed the delay 10ms. It shouldn't matter tbh.